src/charm.py

```diff
@@ -162,7 +168,6 @@ def _configure_sdcore_nms(self, event: EventBase) -> None:
             return
         self._configure_workload()
         self._publish_sdcore_config_url()
-        self._setup_webui_endpoint_url()
```

Comment:
We need to keep this method to set the endpoint URL. It includes the pod ip and it may change.